### ISSUE TITLE
Source fails `make headers_check`

### ISSUE THREAD

**Issue by Ravenslofty** [frno7/linux#35]
```
make ARCH=mipsr5900el-linux-musl -i headers_check 2>&1 | grep error
error: arch/mips/include/uapi/asm/gif.h: missing "WITH Linux-syscall-note" for SPDX-License-Identifier
error: arch/mips/include/uapi/asm/gs.h: missing "WITH Linux-syscall-note" for SPDX-License-Identifier
```

I think you need to modify the SPDX headers to add the "using Linux syscalls is okay" notice that Linux uses.

**Comment by frno7** [comment]
Right. Hmm... I’m not sure these headers ought to be public. The idea is to create a device `/dev/gs` to represent the Graphics Synthesizer as a character device for applications to stream graphics. #10 briefly describes this, as the third option in the description text. Such a device doesn’t exist (yet), so in the meantime, for the initial patch submission in #1, the corresponding headers probably ought to be private.

**Comment by frno7** [comment]
A simple fix is now committed as 1aeb7fdf7a6107d5e7be8b518dcec27127ea854c. Thanks for your report, @Ravenslofty!
